perf/impl/go-libp2p/v0.31/.gitignore

```diff
@@ -0,0 +1,3 @@
+perf
+.cache
+v0.29
```

Comment:
Why v0.29?

Reply:
It was copied from the previous one. It shouldn't be here.